packages/ramda-extension/src/trace.js

```diff
@@ -14,6 +14,6 @@ import { curry } from 'ramda';
  *        compose(calculation2, R_.trace('Page A'), calculation1); // logs "Page A" and result of calculation1
  *
  */
-const trace = curry((x, y) => (console.log(x, y), y));
+const trace = curry(compose(last, tap(apply(console.log)), pair));
```

Comment:
I would prefer more straightforward solution like

`useWith(tap, [curryN(2, console.log), identity])`

Reply:
I would prefer `binary` over `curryN`

Reply:
That's even better :).

Reply:
actually, binary does not work in this case :-(